CMakeLists.txt

```diff
@@ -11,18 +11,15 @@ endif()
 set(VERSION_MAJOR  1)
 set(VERSION_MINOR  0)
 set(VERSION_BUGFIX 0)
-set(VERSION_STRING "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_BUGFIX}")
+set(VERSION_STRING "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_BUGFIX} (${GIT_COMMIT_HASH})")
```

Comment:
Didn't test, but would this use the full 32byte commit hash? What about shorten to 8 or 12 chars? Collision risk would still be very rar.